test/powershell/Language/Scripting/Scripting.Followup.Tests.ps1

```diff
@@ -90,4 +90,43 @@ Describe "Scripting.Followup.Tests" -Tags "CI" {
 
         TestFunc2 | Should -Be @("1", "2")
     }
+
+    It "'[NullString]::Value' should be treated as string type when resolving .NET method" {
+
+        $testType = 'NullStringTest' -as [type]
+        if (-not $testType) {
+            Add-Type -TypeDefinition @"
+    using System;
```

Comment:
Weird indent, why not keep it all the way to the left if you're not aligning to the PowerShell code

Reply:
Move to the very left.